tests/testthat/test-boost_tree-mboost.R

```diff
@@ -50,24 +50,32 @@ test_that("time predictions", {
 test_that("survival predictions", {
   pred_time <- c(0, 100, 200, 10000)
 
-  lung2 <- lung[-14, ]
```

Comment:
this removed a missing value. mboost can fit and predict with missing values so I figured it'd be good to make it do this at least in this test. there is an element of randomness involved though, hence setting the seed a couple of times.